lib/api/bucketGetQuota.js

```diff
@@ -0,0 +1,58 @@
+const { errors } = require('arsenal');
+
+const { standardMetadataValidateBucket } = require('../metadata/metadataUtils');
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+
+/**
+ * bucketGetQuota - Get the bucket quota
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+function bucketGetQuota(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'bucketGetQuota' });
+    const { bucketName, headers, method } = request;
+    const metadataValParams = {
+        authInfo,
+        bucketName,
+        requestType: request.apiMethods || 'bucketGetQuota',
+        request,
+    };
+    const xml = [];
+    xml.push(
+        '<?xml version="1.0" encoding="UTF-8"?>',
+        '<GetBucketQuota>',
+        '<Name>', bucketName, '</Name>',
+    );
+
+    return standardMetadataValidateBucket(metadataValParams, request.actionImplicitDenies, log, (err, bucket) => {
+        const corsHeaders = collectCorsHeaders(headers.origin, method, bucket);
+        if (err) {
+            log.debug('error processing request', {
+                error: err,
+                method: 'bucketGetQuota',
+            });
+            return callback(err, null, corsHeaders);
+        }
+        const bucketQuota = bucket.getQuota();
+            log.debug('Heres the quota', {
+                quota: bucketQuota,
+                method: 'bucketGetQuota',
+            });
+        if (!bucketQuota) {
+            log.debug('error processing request', {
+                error: errors.NoSuchBucketQuota,
+                method: 'bucketGetQuota',
+            });
+            return callback(errors.NoSuchBucketQuota, null,
+                corsHeaders);
+        }
+        xml.push('<Quota>', bucketQuota, '</Quota>',
+            '</GetBucketQuota>');
+        return callback(null, xml.join(''), corsHeaders);
```

Comment:
We should have metrics here as well (pushMetrics)

Reply:
done here: https://github.com/scality/cloudserver/pull/5552/files#diff-cc21292b51ad30d5170e673d7ec74373aafcbe1f07e3d01460156d6294d95433R50